tests/qemu/info.pm

```diff
@@ -22,7 +22,7 @@ sub run {
 
     script_run 'lscpu';
     script_run 'uname -a';
-    script_run "egrep -o '(vmx|svm)' /proc/cpuinfo | sort | uniq";
+    script_run "egrep -o '(vmx|svm|sie)' /proc/cpuinfo | sort | uniq";
```

Comment:
What is `sie`?

Reply:
https://www.ibm.com/docs/en/linux-on-systems?topic=troubleshooting-sie-capability